iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/fileio/FileIOManager.java

```diff
@@ -33,9 +33,16 @@ public class FileIOManager {
   @Qualifier("LocalFileIO")
   FileIO localFileIO;
 
+  @Autowired(required = false)
+  @Qualifier("S3FileIO")
```

Comment:
Can this be:
```
@Autowired(required = false)
S3FileIO s3FileIO;
```
instead of with qualifier, qualifier was introduced only to tell Local vs Hdfs apart

Reply:
Done